tests/libs/__mocks__/prismaMock.ts

```diff
@@ -14,5 +14,5 @@ beforeEach(() => {
   mockReset(prisma);
 });
 
-const prisma = mockDeep<PrismaClient>();
```

Comment:
Weirdly - TS infering this was a massive performance hit. Casting it to the same type seemed to fix some of my issues